### ISSUE TITLE
Unable to upload images with 0.0.3

### ISSUE THREAD

**Issue by dgimb89** [stfnhmplr/pagekit-gallery#39]
I've been unable to upload any image with the gallery extension since the update. I tried to reinstall the extension multiple times - no success. The image is uploaded to the storage (/storage/shw-gallery/1510563171_1.jpg) and the thumbnail (/storage/shw-gallery/thumbnails/tn_1510563171_1.jpg) is created successfully, the cache folder is empty.

Here's the Request Payload for an Image Upload

```
------WebKitFormBoundary803NglEhme8lT9DH
Content-Disposition: form-data; name="images[0]"; filename="1.jpg"
Content-Type: image/jpeg


------WebKitFormBoundary803NglEhme8lT9DH
Content-Disposition: form-data; name="images[length]"

1
------WebKitFormBoundary803NglEhme8lT9DH
Content-Disposition: form-data; name="images[item]"

function item() { [native code] }
------WebKitFormBoundary803NglEhme8lT9DH
Content-Disposition: form-data; name="id"

1
------WebKitFormBoundary803NglEhme8lT9DH--
```

Here's the call trace:
![bildschirmfoto 2017-11-13 um 09 54 13](https://user-images.githubusercontent.com/931787/32717248-32806d98-c859-11e7-95b5-0d24ab99883e.png)

**Comment by BrightdayStudio** [comment]
me too.

**Comment by stfnhmplr** [comment]
Thanks for your feedback. The issue was a custom directory for image caching. Should be fixed with 0.3.1
